content/proposals.md

```diff
@@ -493,6 +493,34 @@ In summary:
     a small table at the bottom mapping the various values from stable
     to unstable.
 
+### Placeholder MSCs
+
+Some proposals may contain security-sensitive or private context which can't be
+publicly disclosed until a later stage in the idea or solution process. Typically,
+the initial idea is validated using some amount of implementation or experimentation
+and may require an MSC number to make that implementation easier.
+
+Placeholder MSCs are used to represent proposals in a state where implementation
+is ongoing, but the MSC details can't yet be disclosed. Authors which feel as
+though their MSC could be highly sensitive MUST get in contact with the Spec Core
+Team or Security Team prior to opening their MSC. If either team determines that
+a placeholder MSC is required, it may be opened as such.
+
+There are a few expectations attached to placeholder MSCs:
+
+* They are tagged as WIP drafts ahead of receiving real content.
```

Comment:
Does this mean "WIP" prefix in the title and "draft" PR state? If we're explicit on this, we could make the bot _not_ announce placeholder PR creation (which, I think, is what occasionally trips people up because it's a notification for something they cannot read).

Reply:
fixed in 8a68e4e67e71bc4f6dc8b1dde53d5cd2627c1b3d